documentation/docs/index.md

```diff
@@ -21,25 +24,44 @@ and an application-level encrypted database indexed with Findex.
     docker run -p 9998:9998 --name kms ghcr.io/cosmian/kms:4.18.0
     ```
 
-    Check the Cosmian KMS server version
+    Pre-built binaries, for both the server and CLI (called `ckms`) are available for multiple 
+    operating systems on [Cosmian packages](https://package.cosmian.com/kms/4.18.0/).
+
+    Using [`ckms`](./cli/cli.md), you can easily manage the server: 
 
+    1) Create a 256-bit symmetric key
+    
     ```sh
-    curl http://localhost:9998/version
+    ckms sym keys create --number-of-bits 256 --algorithm aes --tag my-file-key
```

Comment:
` --number-of-bits 256 --algorithm aes` these are default argument values - omit them to make it simple ?

Reply:
You are right, but I prefer to be explicit.
If not, without further explanation, it looks either magic or hard-coded.